Tests/Engine/CommunityAnalyzerRules/CommunityAnalyzerRules.psm1

```diff
@@ -126,7 +126,7 @@ function Measure-RequiresRunAsAdministrator
 .DESCRIPTION
     The #Requires statement prevents a script from running unless the Windows PowerShell version, modules, snap-ins, and module and snap-in version prerequisites are met.
     From Windows PowerShell 3.0, the #Requires statement let script developers specify Windows PowerShell modules that the script requires.
-    To fix a violation of this rule, please consider to use #Requires -RunAsAdministrator instead of using Import-Module.
+    To fix a violation of this rule, please consider to use #Requires -Modules instead of using Import-Module.
```

Comment:
Thanks for the contribution. The fix of this copy-paste mistake makes sense.
However, by looking at the documentation [here](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_requires), I think it would be better to state the full synax:
`#Requires -Modules { <Module-Name> | <Hashtable> }`